setup.py

```diff
@@ -33,9 +33,9 @@
         "seaborn",
         "numpy",
         "matplotlib",
-        "click",
+        "click==7.0",
         "Pillow",
-        "Jinja2",
+        "Jinja2==2.10.3",
```

Comment:
Hi, I'm wondering why we pin the versions of these packages to these?

Reply:
`2.10.3` is what I have tested and works properly. If you use newer jinja2, this issue may happen to you

```
ImportError: cannot import name 'Markup' from 'jinja2'
```

and the import behavior seems to change since `3.1.0`. See the release note https://jinja.palletsprojects.com/en/3.1.x/changes/#version-3-1-0

Reply:
interesting 🤔 but it might be better for us to create an issue for reminding us to fix it from the root in the future. otherwise, we will not be able to use the newer versions

Reply:
Here you go! https://github.com/pycontw/postevent-report-generator/issues/55